comparators/name_ref_changes.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var MAJOR_ROAD_TYPES = ['motorway', 'trunk', 'motorway_link', 'trunk_link'];
+
+function getHighwayType(feature) {
+  return feature.properties.highway;
+}
+
+function isMajorRoad(feature) {
+  var highwayType = getHighwayType(feature);
+  return MAJOR_ROAD_TYPES.indexOf(highwayType) !== -1;
+}
+
+function nameRefChanged(newVersion, oldVersion) {
+  if (!oldVersion || newVersion.deleted) {
+    return false;
+  }
+
+  if (isMajorRoad(oldVersion)) {
+    if (
+      oldVersion.properties.name !== newVersion.properties.name ||
+      oldVersion.properties.ref !== newVersion.properties.ref
+    ) {
+      return {
+        'result:name_ref_changes': true
```

Comment:
👋 , just a minor one, can also be ignored. The result name should be same as compare function name. Here,
`result:name_ref_changes` can be called as `result:name_ref_changed`. But otherwise, this PR looks good to me.